blocks/blogheader/blogheader.css

```diff
@@ -40,42 +47,190 @@ nav#blogheader li a {
 }
 
-nav#blogheader li {
-  display: table-cell;
-  border-right: 1px solid white;
-  color: white;
-  text-align: center;
-  padding: 1.5rem 1.2rem;
-  width: calc(55rem / var(--number-of-menu-items));
-}
-
 nav#blogheader li.blogsearch-menu-container {
   border: none;
   text-align: right;
   width: auto;
+  padding: 0;
   display: flex;
+  flex-direction: column;
   justify-content: flex-end;
 }
 
-nav#blogheader li.active {
-  background-color: black;
+/* Search Related Styles */
+.blogsearch {
+  width: 100%;
+  display: flex;
+  padding: 1rem;
 }
 
-/* Style for the paragraph */
-nav#blogheader p {
+.blogsearch > form {
+  display: block;
+  padding-right: 10px;
+  height: 2.5rem;
+  width: 100%;
+}
+
+.blogsearch form > div.search-container {
+  position: relative;
+  height: 100%;
+}
+
+.blogsearch form div.search-container i.search-icon {
+  background-image: url("/icons/blog-search-icon.webp");
+  width: 1.5rem;
+  height: 1.5rem;
+  left: 1.3rem;
+  top: 0.7rem;
+  background-repeat: no-repeat;
+  display: block;
+  position: absolute;
+}
+
+.blogsearch form.searching div.search-container i.search-icon {
+  display: none;
+}
+
+.blogsearch form div.search-container span.search-indicator {
+  display: none;
+  position: absolute;
+  left: 1.3rem;
+  top: 0.7rem;
+  border: 3px solid #ccc;
+  border-top-color: #333;
+  border-radius: 50%;
+  width: 1rem;
+  height: 1rem;
+  animation: loading-indicator-spin 2s linear infinite;
+}
+
+.blogsearch form.searching div span.search-indicator {
+     display: block;
+}
+
+/* stylelint-disable */
+@keyframes loading-indicator-spin {
+  0 {
+    transform: rotate(0)
+  }
+  100% {
+    transform: rotate(360deg)
+  }
+}
+/* stylelint-enable */
+
+.blogsearch form div.search-container input {
+  background-color: rgb(255 255 255 / 10%);
+  background-image: none;
+  border-radius: 12px;
+  box-shadow: 0 2px 1px rgb(0 0 0 / 60%) inset, 0 1px 0 rgb(255 255 255 / 20%);
+  padding: 0;
+  padding-left: 3rem;
+  font-family: var(--ff-gilroy-regular);
+  font-size: 1.4rem;
   margin: 0;
+  position: relative;
+  border: 2px inset #293e40;
+  color: #fff;
+  outline: 0;
+  height: 100%;
+  width: 100%;
+  max-width: unset;
+  display: inline-block;
+}
+
+#blogheader div.search-results {
+  display: none;
+  left: 0;
+  top: 0;
+  padding: 1.5rem;
+  padding-bottom: 1rem;
+  max-height: 20rem;
+  min-width: 100%;
+  background: white;
+  border: 1px solid #ccc;
+  color: black;
+  z-index: 100;
+  text-align: left;
+  overflow: scroll;
+}
+
+#blogheader div.search-results a {
+  display: block;
+  color: #293e40;
+  -webkit-font-smoothing: antialiased;
+  letter-spacing: normal;
+  line-height: 1.6rem;
+  margin-bottom: 1rem;
+  font-weight: normal;
+  font-family: var(--ff-gilroy-regular);
+  font-size: 1.4rem;
+  border: none;
   padding: 0;
 }
 
+#blogheader div.search-results a:hover {
+  text-decoration: underline #81b5a1;
+}
+
+#blogheader div.search-results a mark {
+  background-color: #81b5a1;
```

Comment:
Please try to use the CSS variables where they are available: `var(--link-background-color)`. This would make it easier for us if we need to replace colors.

Reply:
will do!

Reply:
fixed in 7fc729a